build_daemon/lib/src/daemon.dart

```diff
@@ -21,6 +21,14 @@ String runningVersion(String workingDirectory) {
   return versionFile.readAsStringSync();
 }
 
+/// Returns the current options of the running build daemon. Null if one isn't
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph